Dockerfile

```diff
@@ -0,0 +1,10 @@
+from golang:1.15-alpine
+
+WORKDIR /go/src/app
+
+COPY . .
```

Comment:
Copy `go.mod` and `go.sum` before copying the code, and run `go mod install`. Then you don't need to do the `go get` and `go install`.

Reply:
sure, that's the best idea. 

By the way, `COPY go.mod go.sum ./` works.